src/transformers/dynamic_module_utils.py

```diff
@@ -432,8 +433,9 @@ def get_class_from_dynamic_module(
         force_download (`bool`, *optional*, defaults to `False`):
             Whether or not to force to (re-)download the configuration files and override the cached versions if they
             exist.
-        resume_download (`bool`, *optional*, defaults to `False`):
-            Whether or not to delete incompletely received file. Attempts to resume the download if such a file exists.
+        resume_download (defaults to `None`):
```

Comment:
```suggestion
        resume_download (`bool`, *optional*):
```

Reply:
> I just put in suggestions to make the docstrings in the "transformers format", specifically we don't say anything about the default if it's None

My goal was specifically not to document which type is expected since it's deprecated and no matter the value you set, it will be ignored. So it's not a `bool` value, it's a `Any` value. That's why I've put just `resume_download (defaults to `None`):`. 

Would you be ok to set it to `resume_download:` otherwise? (i.e. no type annotation at all)

Reply:
Yep! Just `resume_download:` works for me!

Reply:
Done in https://github.com/huggingface/transformers/pull/30620/commits/2eba9af272aa987fee1cae9278967440b1fbdf10.